### PR TITLE
chore: fix debugger launcher on window

### PR DESCRIPTION
**Pull request by h-a-n-a**
## Summary

Fix debugger launcher on windows

## Require Documentation?

- [x] No
- [ ] Yes, the corresponding rspack-website PR is \_\_

### CODE REVIEW

File: scripts/tsconfig.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
 
     /* JavaScript Support */
     "allowJs": true /* Allow JavaScript files to be a part of your program. Use the 'checkJS' option to get errors from these files. */,
-    "checkJs": true /* Enable error reporting in type-checked JavaScript files. */,
+    "checkJs": false /* Enable error reporting in type-checked JavaScript files. */,
```
Comment (hardfist) [inline scripts/tsconfig.json:48]: why change this?

Comment (h-a-n-a) [reply scripts/tsconfig.json:48]: I don't think it's necessary for JS files...

Comment (h-a-n-a) [reply scripts/tsconfig.json:48]: Actually, we didn't do type checking on the CI anyway and it make ts server complaining. So I turned it off.

Comment (hardfist) [reply scripts/tsconfig.json:48]: does not it provide autocomplete

Comment (h-a-n-a) [reply scripts/tsconfig.json:48]: Yes, it does ;--)

**Review by hardfist — APPROVED** [review]

**Pull request merged**